test/hotspot/jtreg/compiler/arraycopy/TestArrayCopyConjoint.java

```diff
@@ -262,10 +261,6 @@ public static void main(String [] args) {
           reinit(long.class);
           testLong_constant_LT64B (index , index+2);
           reinit(long.class);
-        }
-        System.out.println("PASS : " + validate_ctr);
-      } catch (Exception e) {
-        System.out.println(e.getMessage());
```

Comment:
`try/catch` is there because `arraycopy` may throw exception due to some mess up in code generation and we want call stack and information about that to debug the issue.

Reply:
But with that try/catch, no exception will be thrown and therefore the test would pass, right? And the exception would contain a stack trace, no need to catch it and print its contents.

Reply:
You are right. In this case, please also update `TestArrayCopyDisjoint.java` test.
It could be "copy/paste" issue when these tests were created based on other existing tests.
May be in separate RFE we should review tests which use `catch` and may hide a failure.

Reply:
Thanks for the idea, I applied it to `TestArrayCopyDisjoint.java`, and updated the description above.

Reply:
Filed follow-up RFE
https://bugs.openjdk.org/browse/JDK-8302135